pages/person/_itemId/index.vue

```diff
@@ -26,27 +26,23 @@
           </v-col>
           <v-col cols="3">
             <v-row v-if="birthDate">
-              <v-col cols="3" class="text--secondary">Birth</v-col>
+              <v-col cols="3" class="text--secondary">{{ $t('birth') }}</v-col>
               <v-col cols="9">
                 {{ birthDate }}
               </v-col>
             </v-row>
             <v-row v-if="deathDate">
-              <v-col cols="3" class="text--secondary">Death</v-col>
+              <v-col cols="3" class="text--secondary">{{ $t('death') }}</v-col>
               <v-col cols="9">
                 {{ deathDate }}
               </v-col>
             </v-row>
             <v-row v-if="birthPlace">
-              <v-col cols="3" class="text--secondary">Birthplace</v-col>
-              <v-col cols="9">
-                {{ birthPlace }}
+              <v-col cols="3" class="text--secondary">
+                {{ $t('Birthplace') }}
```

Comment:
```suggestion
                {{ $t('item.person.birthplace') }}
```

Also first letter shouldn't be capitalized